### DISCUSSION TITLE
Test du premier module Matter miniR4M de Sonoff - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#21] [Comments]
# Test du premier module Matter miniR4M de Sonoff - Haade.fr

En avant première je vais tester le premier module Matter de Sonoff le miniR4M et une intégration dans Home Assistant

https://haade.fr/fr/blog/test-du-premier-module-mini-interrupteur-sonoff-minir4m-matter

**Comment by Seyko91** [comment]
Bonjour,
Merci pour le test, tres intéressant.
Je suis en plein domotisation de mon chauffage électrique (radiateur elec), je me demande si ce module peut être utilisé comme le zbmini + diode pour contrôler un radiateur via fil pilote ? Le but étant purement d'avoir le protocole matter plutôt que zigbee.

Merci pour ton retour.
